versions.props

```diff
@@ -23,6 +23,7 @@ org.assertj:assertj-* = 3.5.2
 org.codehaus.groovy:groovy-all = 2.4.4
 org.hamcrest:* = 1.3
 org.hibernate:hibernate-validator = 5.1.3.Final
+org.immutables:* = 2.2.8
```

Comment:
How about removing `immutables` from `versions.gradle` or atleast adding a comment to prevent anyone from  adding the previous version in future.